src/stores/user.ts

```diff
@@ -0,0 +1,72 @@
+import { defineStore } from "pinia";
+import router from "@/routes";
+
+interface IUser {
+  email: string;
+  firstname: string;
+  lastname: string;
+  password: string;
+}
+
+export const useUserStore = defineStore({
+  id: "user",
+  state: () => ({
+    messageLogin: "",
+    messageSignup: "",
+    isLogged: false,
+    userBank: [
+      {
+        email: "admin@admin.admin",
+        firstname: "admin",
+        lastname: "admin",
+        password: "admin123",
+      },
+    ],
+    currentUser: {
+      email: "",
+      firstname: "",
+      lastname: "",
+      password: "",
+    },
+  }),
+  getters: {
+    userCount: (state) => state.userBank.length,
+    showUsers: (state) => console.log(state.userBank),
+    loggedStatus: (state) => state.isLogged,
+  },
+  actions: {
+    addPerson(user: IUser) {
+      this.userBank.push(user);
+    },
+
+    authUser(email: string, password: string) {
+      //check if email& password matches
```

Comment:
// check if email & password matches

fix spacing
```suggestion
      //check if email& password matches
```